src/main/java/ch/uzh/ifi/seal/soprafs20/entity/Game.java

```diff
@@ -41,8 +43,12 @@
     @Column(nullable = false)
     private int wordIndex;
 
+    @Column
+    @ElementCollection
+    private List<Integer> lastWordIndex = new ArrayList<>();
+
     @Column(nullable = false)
-    private int score;
+    private int roundScore;
```

Comment:
changed the variable to roundScore to get the score for each round and this will be resetted during the endRound.
This roundScore will be saved to score arrayList at the time of reset